packages/flutter/lib/src/widgets/transitions.dart

```diff
@@ -307,14 +307,20 @@ class SlideTransition extends AnimatedWidget {
 /// above:
 ///
 /// ```dart
-/// late AnimationController _controller = AnimationController(
```

Comment:
I believe all you need is a final here:

```dart
/// late final AnimationController _controller = AnimationController(
```

Reply:
But still it is give the error on "vsync:this"